project2-beat-that/script.js

```diff
@@ -1,4 +1,272 @@
+// Generate 2 dice rolls for player 1
+// > return user the 2 dice rolls
+// > Allow player 1 to choose order of 2 dice rolls
+// > return the player's choice
+
+// 2nd version
+
+var comparingCombineDiceRolls = function () {
+  var max = Math.max.apply(null, playerScores);
+  var winningPlayer = playerScores.indexOf(max);
+  return winningPlayer;
+};
+
+var randomDiceRolls = function () {
+  return Math.floor(Math.random() * 6) + 1;
+};
+
+var diceRoll1 = 0;
+var diceRoll2 = 0;
+
+var rollState = true;
+var playerNumber = 0;
+var numPlayers = 2;
+var playerScores = Array(numPlayers);
+playerScores.fill(0);
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  // If supposed to roll
+  if (rollState === true) {
+    // If user gives non-empty input, return error msg
+    if (input !== "") {
+      return 'Please roll the dice by clicking "submit"';
+    }
```

Comment:
this works, but maybe we can still progress with the roll even if the user submits some input. just gotta ignore the input?